tests/integration/helm_prometheus_metrics_test.go

```diff
@@ -14,7 +14,7 @@ func Test_Helm_Prometheus_Metrics(t *testing.T) {
 	// defaults without otel metrics collector metrics, but with Prometheus metrics
 	expectedMetricsGroups := make([][]string, len(internal.DefaultExpectedMetricsGroups))
 	copy(expectedMetricsGroups, internal.DefaultExpectedMetricsGroups)
-	expectedMetricsGroups = append(expectedMetricsGroups, internal.PrometheusMetrics, internal.DefaultOtelcolMetrics)
+	expectedMetricsGroups = append(expectedMetricsGroups, internal.PrometheusMetrics, internal.DefaultOtelcolMetrics, internal.BucketMetrics)
```

Comment:
I'm not convinced to this change. as we filter out all buckets in metadata layer, it shouldn't be needed. Maybe telegraf receiver do not create histogram metrics 🤔